pootle/apps/pootle_project/models.py

```diff
@@ -339,13 +337,16 @@ def resources(self):
         '''
         cursor = connection.cursor()
         cursor.execute(sql_query, [resources_path, resources_path])
-
         results = cursor.fetchall()
 
         # Flatten tuple and sort in a list
-        resources = list(reduce(lambda x,y: x+y, results))
-        resources.sort(key=get_path_sortkey)
+        resources = set()
+        for result in results:
+            s = result[0]
```

Comment:
You mean compare to list(reduce(rocket(scientist(coder())))) ;)

Its a temporary var that is a string.  Giving it a sensible name like 'path' just makes the second line less readable... trust me I tried.